src/bundle/Resources/config/services.yml

```diff
@@ -4,6 +4,7 @@ imports:
     - { resource: services/tabs.yml }
     - { resource: services/menu.yml }
     - { resource: services/pagination.yml }
+    - { resource: services/ui_config_mappers.yml }
```

Comment:
If there are multiple files with the same prefix I'd put them into new directory because as you can see it quickly gets out of hands.

Reply:
> If there are multiple files with the same prefix I'd put them into new directory because as you can see it quickly gets out of hands.

Refactored as we discussed.